core/adt/balance_table.cpp

```diff
@@ -14,21 +14,21 @@ OUTCOME_CPP_DEFINE_CATEGORY(fc::adt, BalanceTableError, e) {
 }
 
 namespace fc::adt {
-  outcome::result<TokenAmount> BalanceTable::get(const Key &key) {
-    OUTCOME_TRY(value, tryGet(key));
-    if (value) {
-      return *value;
-    }
-    return 0;
-  }
-
```

Comment:
Why it was removed?

Reply:
In previous PR I introduced `BalanceTable` as a wrapper for `Map`, and because I thought that `BalanceTableHamt.get` behavior was wrong.

But I was mistaken while reading [balancetable.go](https://github.com/filecoin-project/specs-actors/blob/master/actors/util/adt/balancetable.go#L38).

I removed this new method, so it will fallback to `Map.get`.